docs/tutorials/gradients.ipynb

```diff
@@ -126,6 +126,7 @@
       },
       "source": [
         "try:\n",
+         "    # %tensorflow_version only works in Colab.\n",
```

Comment:
Extra space here

Reply:
@MichaelBroughton I've corrected my formatting, kindly review it.